org.corluma.Corluma.yml

```diff
@@ -0,0 +1,16 @@
+app-id: org.corluma.Corluma
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: corluma
+finish-args: [--share=network,--share=ipc,--socket=x11,--socket=wayland,--filesystem=host,--device=dri]
+modules:
+  - name: corluma
+    buildsystem: simple
+    build-commands:
+      - install -D corluma /app/bin/corluma
```

Comment:
so you install a binary and still build for both aarch64 and x86_64. How is this gonna work?

Reply:
also are you allow to redistribute this? It doesn't look that way to me.

Reply:
reduced scope to x86_64. 

I am the sole maintainer, and the owner of https://corluma.net. I give myself rights to distribute it. As initial proof, I modified https://corluma.net/linux to mention an upcoming flatpak release. As secondary proof, v0.35.14 was made available for Ubuntu yesterday and is not on any app stores yet (but will likely go up tomorrow). Let me know if I need to provide any more proof.

Reply:
why not a version fo aarch64?

Reply:
planned for future release, was out of scope originally due to the maintenance nightmare of supporting multiple distros in aarch64, but flatpak removes that nightmare.